fabric-networking-api-v1/src/client/java/net/fabricmc/fabric/api/client/networking/v1/ClientPlayNetworking.java

```diff
@@ -255,4 +403,44 @@ public interface PlayChannelHandler {
 		 */
 		void receive(MinecraftClient client, ClientPlayNetworkHandler handler, PacketByteBuf buf, PacketSender responseSender);
 	}
+
+	/**
+	 * An internal packet handler that works as a proxy between old and new API.
+	 * @param <T> the type of the packet
+	 */
+	interface PlayChannelHandlerProxy<T extends FabricPacket> extends PlayChannelHandler {
```

Comment:
```suggestion
	private interface PlayChannelHandlerProxy<T extends FabricPacket> extends PlayChannelHandler {
```
I think this can be private?